IOMC/Input/src/MCFileSource.cc

```diff
@@ -32,7 +33,8 @@ MCFileSource::MCFileSource(const ParameterSet & pset, InputSourceDescription con
   }  
   
   reader_->initialize(fileName);  
-  produces<HepMCProduct>();
+  produces<HepMCProduct>("generator");
```

Comment:
if I remember correctly, if otherwise the following stage (SIM) gives an error because expects "generator"
@amarini can you please comment?